hathor/util.py

```diff
@@ -759,3 +763,33 @@ def is_token_uid_valid(token_uid: bytes) -> bool:
         return True
     else:
         return False
+
+
+@dataclass
+class EnvironmentInfo:
+    # Changing these names could impact logging collectors that parse them
+    python_implementation: str
+    hathor_core_args: str
+    hathor_core_version: str
+    peer_id: Optional[str]
+    network: str
+
+    def as_dict(self):
+        return asdict(self)
+
+
+def get_environment_info(args: Namespace, peer_id: Optional[str]) -> EnvironmentInfo:
+    environment_info = EnvironmentInfo(
+        python_implementation=str(sys.implementation),
+        hathor_core_args=str(args),
+        hathor_core_version=get_hathor_core_version(),
+        peer_id=peer_id,
+        # We want to ignore the testnet suffixes here. "testnet-golf" should be reported only as "testnet".
+        network=settings.NETWORK_NAME.split("-")[0]
```

Comment:
Shouldn't we add both the suffix (e.g., mainnet and testnet) and the full network?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/490/commits/33b1b7b113d4cd61dd9f40c9609258f400f3fb8a

Reply:
Thanks!